nautobot/docs/release-notes/version-1.5.md

```diff
@@ -10,6 +10,17 @@ If you are a user migrating from NetBox to Nautobot, please refer to the ["Migra
 
 ### Added
 
+#### Required Relationships ([#873](https://github.com/nautobot/nautobot/issues/873))
```

Comment:
Historically we've alphabetized the list of headers in the Release Overview section, in which case this would belong after `Nestable LocationTypes`.